### DISCUSSION TITLE
USDHelper equivalent for Scene URDFs

### DISCUSSION THREAD

**Discussion by pushkalkatara** [NVlabs/curobo#195] [Software Q&A] [answered]
Hi, Thanks for the amazing library on motion planning!

I am trying to use CuRobot with other simulators, specifically IssacGym. 

I have a scene URDF and want to load this scene into CuRobot WorldConfig. What steps should I follow for this conversion?

For IssacSim, `UsdHelper` is provided, is there a specific tool for IssacGym or URDF/XML scenes?

**Comment by balakumar-s** [comment, marked as answer]
We do not have a world parsing API for other simulators.

You can load the scene as a WorldConfig: https://curobo.org/_api/curobo.geom.types.html#curobo.geom.types.WorldConfig.from_dict

To update the pose of objects, you can use this: https://curobo.org/_api/curobo.geom.sdf.world.html#curobo.geom.sdf.world.WorldPrimitiveCollision.update_obstacle_pose
